### PR TITLE
Yaw issue72 update profile

### LINKED ISSUE #72: User Profile Update Page
Page Url: /profile/update
Module: Auth - Profile

Authority:
- User themselves

Usage
Edit own Profile Information

- [x] Development

### PR DESCRIPTION
**Pull request by HowardYaw**
resolved #72 

Pre-requisite: 
Merge #120 

**Update User Profile Page**
image

### CODE REVIEW

**Review by LUXIANZE — DISMISSED** [review]

File: src/main/webapp/app/modules/user-profile/complete-user-profile.tsx [hunk 1]
```diff
@@ -59,6 +60,7 @@ class CompleteUserProfile extends React.Component<ICompleteUserProfileProps, {}>
               <p>Loading...</p>
             ) : (
               <AvForm model={userProfile} onSubmit={this.completeProfile}>
+                {window.console.log(userProfile)}
```
Comment (CarlosSia) [inline src/main/webapp/app/modules/user-profile/complete-user-profile.tsx]: this one is for debugging?

Comment (kachun333) [reply src/main/webapp/app/modules/user-profile/complete-user-profile.tsx]: @HowardYaw I once noticed you wrote `window.console.log` for other pages as well. Is it intentional to log it to the console?

Comment (HowardYaw) [reply src/main/webapp/app/modules/user-profile/complete-user-profile.tsx]: @kachun333 which page?

Comment (HowardYaw) [reply src/main/webapp/app/modules/user-profile/complete-user-profile.tsx]: @CarlosSia removed on 9022f20616347651d8f9daf441652f7f22ec6849

**Review by CarlosSia — APPROVED** [review]

**Review by LUXIANZE — APPROVED** [review]

**Pull request merged**